tests/langchain/chain.py

```diff
@@ -10,7 +10,7 @@
 
 from mlflow.models import ModelConfig, set_model
 
-base_config = ModelConfig(development_config="tests/langchain/config.yaml")
+base_config = ModelConfig(development_config="tests/langchain/config.yml")
```

Comment:
really not sure how this test used to work because ".yaml" doesn't exist

Reply:
some magic from me 🙈

Reply:
just realized that this isn't used right now (only needed to change this because I messed something else up). leaving it corrected though